host/frontend/host-orchestrator/operation_test.go

```diff
@@ -17,37 +17,32 @@ package main
 import (
 	"errors"
 	"testing"
+	"time"
 )
 
 func TestMapOMNewOperation(t *testing.T) {
 	opName := "operation-1"
-	uuidFactory := func() string { return opName }
-
-	t.Run("succeeds", func(t *testing.T) {
-		om := NewMapOM(uuidFactory)
-		expectedOp := OperationData{
-			Name: opName,
-			Done: false,
-		}
+	om := NewMapOM(func() string { return opName })
+	expectedOp := Operation{
+		Name: opName,
+		Done: false,
+	}
 
-		op, _ := om.NewOperation()
+	op := om.New()
 
-		if op != expectedOp {
-			t.Errorf("expected <<%+v>>, got %+v", expectedOp, op)
-		}
-	})
+	if op != expectedOp {
```

Comment:
comparing the entire struct here has the same problem as elsewhere: If you add fields to the struct the test will start failing.

Reply:
changed tests TestMapOMGetOperation and TestMapOMCompleteOperation following this suggestion: https://github.com/google/android-cuttlefish/pull/126/commits/7e98479450f2daf884bf4916d1376eaa87d8f7ae

However, in this specific test case, I think it's best to make sure what a brand new operation fully contains, even though it means that we need to update the test in case we decide to add new information when creating a new information.

Reply:
changed the test checking for the relevant fields this time: the name is not empty, done is false and result is empty.